modules/files/files.sh

```diff
@@ -1,38 +1,54 @@
 #!/usr/bin/env bash
 
 # Tell build process to exit if there are any errors.
-set -euo pipefail
+set -oue pipefail
 
-get_yaml_array FILES '.files[]' "$1"
+get_yaml_array ADD_FILES '.add[]' "$1"
```

Comment:
```suggestion
get_yaml_array ADD_FILES '.files[]' "$1"
```

Don't break existing configs.

Reply:
Optionally, if we want to rename this, we could support both `files:` and `copy:`. I think `copy:`  is better than `add:`.

Reply:
And the shell variable maybe should be `COPY_FILES` as well.